markup.go

```diff
@@ -144,13 +143,24 @@ type ClassMap map[string]bool
 
 // Apply implements the Markup interface.
 func (m ClassMap) Apply(h *HTML) {
-	var classes []string
+	if h.classes == nil {
+		h.classes = make(map[string]bool)
+	}
 	for name, active := range m {
-		if active {
-			classes = append(classes, name)
-		}
+		h.classes[name] = active
 	}
-	Property("className", strings.Join(classes, " ")).Apply(h)
+}
+
+// Class returns markup that applies all given classes to an element
+func Class(classes ...string) Markup {
```

Comment:
Overall I like this change, but it should be implemented differently. We don't need `className` to be a part of `*HTML` as I said in my other comment above; I will send a separate PR to implement this now (this change also should not be a part of this PR)